guides/doc-Content_Management_Guide/topics/Importing_Content.adoc

```diff
@@ -102,11 +102,14 @@ endif::[]
 
 .Procedure
 
-. In the {Project} web UI, navigate to *Content* > *Products* and select the product that you want to use, and then click *Create Repository*.
+. In the {Project} web UI, navigate to *Content* > *Products* and select the product that you want to use, and then click *New Repository*.
 . In the *Name* field, enter a name for the repository.
 {ProjectNameX} automatically completes the *Label* field based on what you have entered for *Name*.
 . From the *Type* list, select the type of repository.
 You can select either a repository for RPM files (`yum`), Docker images (`docker`), Files (`file`) or other.
+. If `yum` content type is selected, you can restrict whether the repository is available to a host based on the host's architecture and OS version.
+.. Optional: From the *Restrict to Architecture* list, select the architecture. The repository will be enabled on content hosts with the selected architecture. Select *No restriction*, which is the default value, to make the repository available to all hosts regardless of the architecture.
+.. Optional: From the *Restrict to OS Version* list, select the OS version. The repository will be enabled on content hosts with the selected OS version. Select *No restriction*, which is the default value, to make the repository available to all hosts regardless of the OS version.
```

Comment:
```suggestion
. Optional: From the *Restrict to OS Version* list, select the OS version. Select *No restriction*, which is the default value, to make the repository available to all hosts regardless of the OS version.
```

Reply:
Now I read this back @suyogsainkar shouldn't it say something like 
```suggestion
.. Optional: From the *Restrict to OS Version* list, select the OS version. The repository will be enabled on content hosts with the selected OS version. Ensure that *No restriction*, which is the default value, is selected to make the repository available to all hosts regardless of the OS version.
```

If it's the default, you do nothing, just check?

Reply:
Updated, thanks.